tests/unit/s2n_session_ticket_test.c

```diff
@@ -372,36 +372,32 @@ int main(int argc, char **argv)
         EXPECT_SUCCESS(s2n_config_set_wall_clock(server_config, mock_time, &mock_current_time));
 
         /* Add one ST key */
-        EXPECT_SUCCESS(s2n_config_add_ticket_crypto_key(server_config, ticket_key_name1, s2n_array_len(ticket_key_name1), ticket_key1, s2n_array_len(ticket_key1), 0));
+        EXPECT_SUCCESS(s2n_config_add_ticket_crypto_key(server_config, ticket_key_name2, s2n_array_len(ticket_key_name2), ticket_key2, s2n_array_len(ticket_key2), 0));
 
         /* Add a mock delay such that key 1 moves to decrypt-only state */
         mock_current_time += server_config->encrypt_decrypt_key_lifetime_in_nanos;
 
         uint32_t key_intro_time = mock_current_time / ONE_SEC_IN_NANOS;
-        EXPECT_SUCCESS(s2n_config_add_ticket_crypto_key(server_config, ticket_key_name2, s2n_array_len(ticket_key_name2),
-                ticket_key2, s2n_array_len(ticket_key2), key_intro_time));
+        EXPECT_SUCCESS(s2n_config_add_ticket_crypto_key(server_config, ticket_key_name1, s2n_array_len(ticket_key_name1),
+                ticket_key1, s2n_array_len(ticket_key1), key_intro_time));
```

Comment:
If possible, it might be good to assert that s2n-tls is capable of sending a new session ticket. This was tested implicitly before because s2n-tls _did_ send a new session ticket, and we asserted on that. But now we're kind of hoping the test is set up properly so that s2n-tls could send a new session ticket but chooses not to, rather than it not being capable of sending a session ticket in the first place.

Could we maybe assert that `s2n_config_is_encrypt_key_available()` is successful or something?